From 2924caf52b020aac58e62560ba2c127c409c646a Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 18 Jun 2020 10:03:10 +0300
Subject: [PATCH] [SW-0] preboot: modify auto generated file rule

This patch removes PREBOOT_SRCS_S in AUTO_GENERATED_FILE rule as
its no more required.

Change-Id: I9a41700e6dcc27d63c357e828a713d2ec2fbcd67
---

diff --git a/Makefile b/Makefile
index 6ac61aa..e9ee0e5 100644
--- a/Makefile
+++ b/Makefile
@@ -71,7 +71,7 @@
 ppboot: $(PPBOOT_OBJECTS) $(PPBOOT_LD_SCRIPT)
 	@$(CC) $(LDFLAGS) -T $(PPBOOT_LD_SCRIPT) -o $(BINDIR)/$@ -Wl,--start-group $(PPBOOT_OBJECTS) $(LDLIBS) -Wl,--end-group
 
-$(AUTO_GENERATED_FILE): localversion-habanalabs $(PREBOOT_SRCS_C) $(PREBOOT_SRCS_S)
+$(AUTO_GENERATED_FILE): localversion-habanalabs $(PREBOOT_SRCS_C)
 	@( printf '#define PREBOOT_VER "%s"\n' '$(shell cut -d- -f2- localversion-habanalabs)' ) > $@
 	@LANG=en_us date +'#define PREBOOT_DATE "%b %d %Y"' >> $@
 	@date +'#define PREBOOT_TIME "%T"' >> $@
